src/core/jpx.js

```diff
@@ -1125,6 +1125,11 @@ var JpxImage = (function JpxImageClosure() {
             zeroBitPlanesTree = new TagTree(width, height);
             precinct.inclusionTree = inclusionTree;
             precinct.zeroBitPlanesTree = zeroBitPlanesTree;
+            for (var l=0; l < layerNumber; l++) {
```

Comment:
Nit: there's still not enough whitespace around all operators. Please add spaces before *and* after `=`.